dist/mix-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-    "/livewire.js": "/livewire.js?id=b4b27f4e53118a0cdaf3"
+    "/livewire.js": "/livewire.js?id=8317e62ea8d2c2ca1a87"
```

Comment:
I couldnt find any docs for how/when the `dist` folder should be updated, ive assumed if changing js files you need to run `npm run prod` and commit the `dist` files as ive done here.

Reply:
yes, that's right for now